kpatch-build/create-diff-object.c

```diff
@@ -742,7 +743,8 @@ static bool kpatch_line_macro_change_only(struct kpatch_elf *kelf,
 			    !strcmp(rela->sym->name, "___might_sleep") ||
 			    !strcmp(rela->sym->name, "__might_fault") ||
 			    !strcmp(rela->sym->name, "printk") ||
-			    !strcmp(rela->sym->name, "lockdep_rcu_suspicious")) {
+			    !strcmp(rela->sym->name, "lockdep_rcu_suspicious") ||
+			    !strcmp(rela->sym->name, "__btrfs_abort_transaction")) {
```

Comment:
**fs/btrfs/ctree.h** also contains **btrfs_handle_fs_error** and **btrfs_panic** macros that similarly pass `__LINE__` to a function as the third parameter.  The latter two count up to the same number of affected .c files as the first, would they be worth handling, too?
```
$ grep -R -o -e btrfs_abort_transaction fs/btrfs | sort -u | wc -l
21
$ grep -R -o -e btrfs_handle_fs_error fs/btrfs | sort -u | wc -l
15
$ grep -R -o -e btrfs_panic fs/btrfs | sort -u | wc -l
5
```

Reply:
Good call, I'll add `__btrfs_handle_fs_error` and `__btrfs_panic`.